### ISSUE TITLE
 Sway 1.8-3 and/or wlroots 0.16.1-2 break AMD multigpu for Radeon 6650 XT and 6800 XT

### ISSUE THREAD

**Issue by dlpullen** [swaywm/sway#7363]
### Please fill out the following:
- **Sway Version:**
  - `sway version 1.8`

- **Debug Log:**
  - https://gist.github.com/dlpullen/3e80463148d4ff33393fd901da8be330#file-sway_debug-log

- **Configuration File:**
  - Default Config. For reference, the default config provided in my install is below:
  - https://gist.github.com/dlpullen/3e80463148d4ff33393fd901da8be330#file-config

- **Stack Trace:**
  - This is only needed if sway crashes.
  - Sway does not crash

- **Description:**
  - The steps you took in plain English to reproduce the problem:
    - Have two AMD GPUs (Radeon 6650 XT, Radeon 6800 XT) active on system using AMDGPU open source driver
    - Start sway with the default config file from /etc/sway/config on my install.
    - Monitors attached to secondary GPU will not work and error messages relating to the outputs are logged in provided debug log

-**Additional Information:**
  -  Using Arch Linux
  -  Problem did not happen with wlroots-0.15.1-6 and sway-1:1.7-10
  -  Problem began when upgrading to wlroots-0.16.1-2 and sway-1:1.8-3

**Comment by emersion** [comment]
I believe this is a duplicate of https://github.com/swaywm/sway/issues/7352

**Comment by rnestler** [comment]
I have a Laptop with an AMD GPU and in my case the screens just stay black. I'm not sure if it is related to this issue though.

**Comment by dlpullen** [comment]
I took a look at that issue you linked and the workarounds. It does look like the same issue. I'm able to get the second GPU working again with either WLR_EGL_NO_MODIFIERS=1 (as identified in the directly linked issue) or WLR_RENDERER=vulkan (in the wlroots issue links in the sway issue you linked). Though, performance seems sporadic when using the vulkan rendering.

Thank you @emersion!

**Comment by dlpullen** [comment]
@emersion, I do have one question about the work around from my review of the issue for wlroots.

You said "A possible workaround is WLR_EGL_NO_MODIFIERS=1, which should force linear for multi-GPU buffers (at the cost of disabling explicit modifiers)." in the wlroots issue. What is the implication or impact of disabling explicit modifiers? I don't see that discussed in the other issue or on the wlroots issue.
